addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -170,5 +182,22 @@ weSnippetEditor.Class.include({
             tab: this.tabs.THEME,
         });
     },
+    /**
+     * Called when search input value changed > adapts the snippets grid.
+     */
+    _onSearchFilterInputChange: function () {
+        var self = this;
+        self._searchFilterValue = self.$('.o_snipet_search_filter_input').val().toLowerCase();
+        // show all snippets
+        self.$('.o_panel').show();
+        self.$('.oe_snippet').filter(function(){
+            var $snippet = $(this);
+            $snippet.toggle(
+                $snippet.attr('name').toLowerCase().indexOf(self._searchFilterValue) > -1
+            );
+        });
+        // clean editor 
+        $('.o_panel_body:not(:has(.oe_snippet:visible))').parent().hide();
```

Comment:
Same: do not use `hide`, also do not seach the whole DOM, seach inside of the menu thanks to `this.$(...`